Makefile

```diff
@@ -44,6 +45,7 @@ glideup:
 build:
 	${DEV_ENV_PREFIX} -e CGO_ENABLED=0 ${DEV_ENV_IMAGE} go build -a -installsuffix cgo -ldflags ${LDFLAGS} -o ${BINARY_DEST_DIR}/boot boot.go || exit 1
 	@$(call check-static-binary,$(BINARY_DEST_DIR)/boot)
+	goupx ${BINARY_DEST_DIR}/boot
```

Comment:
note that this requires devs to install `goupx`. Maybe should this be added into `go-dev-env` instead?